src/flaskwebgui.py

```diff
@@ -7,6 +7,9 @@
 import subprocess
 import socketserver
 import multiprocessing
+import win32gui
+import win32con
+import pygetwindow

```

Comment:
For Mac/Linux users this will cause an import error. These are windows specific packages.